standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -3814,6 +3815,74 @@ public CompactionResponse compact(CompactionRequest rqst) throws MetaException {
     }
   }
 
+  @Override
+  @RetrySemantics.Idempotent
```

Comment:
This means, we can resubmit without issues in case of a failure.
Would it cause issues if we have multiple compactions scheduled for the same table?

Reply:
changed to safeToRetry, checked - no issues